examples/seismic/tti/operators.py

```diff
@@ -36,58 +45,58 @@ def ForwardOperator(model, u, v, src, rec, data, time_order=2,
         # Derive stencil from symbolic equation
         Gyp = (ang3 * u.dx - ang2 * u.dyr)
         Gyy = (first_derivative(Gyp * ang3,
-                                dim=x, side=centered, order=spc_brd,
+                                dim=x, side=centered, order=space_order,
```

Comment:
This has nothing to do with this PR per se, but I think the interface to take a derivative should be simplified. Here it's taking up three lines !
Shouldn't parameters like ``dim, side, ...`` always be specified? if so, why not making this explicit rather than using key-value pairs (i.e., ``first_derivative(Gyp*ang3, x, centered, space_order,)``)